app/controllers/users_controller.rb

```diff
@@ -11,10 +11,13 @@ def show
   end
 
   def requests_not_confirmed
+    @pending_friends = current_user.pending_friends
+    @inverse_friends = current_user.inverse_friends
+    @pending_friendships = current_user.pending_friendships
     @inverse_friendships = current_user.inverse_friendships
   end
 
   def friends
-    @friends = current_user.friends
+    @friends = current_user.friends.include?
   end
```

Comment:
- [ ] I get this error when I try to view my friends' list:
```
ArgumentError in UsersController#friends
wrong number of arguments (given 0, expected 1)
```
It can be fixed by saying:
`@friends = current_user.friends`